frontend/test/__support__/server/collections.ts

```diff
@@ -0,0 +1,49 @@
+import _ from "underscore";
+import type { Scope } from "nock";
+
+import { ROOT_COLLECTION } from "metabase/entities/collections";
+
+import type { Card, Collection } from "metabase-types/api";
+
+import {
+  convertSavedQuestionToVirtualTable,
+  getCollectionVirtualSchemaName,
+  SAVED_QUESTIONS_VIRTUAL_DB_ID,
+} from "metabase-lib/metadata/utils/saved-questions";
+
+export function setupCollectionEndpoints(
+  scope: Scope,
+  collections: Collection[],
+) {
+  scope.get("/api/collection").reply(200, collections);
```

Comment:
could this file maybe be called `collection.ts` to make it correspond with the endpoints it mocks?